package.json

```diff
@@ -486,6 +486,7 @@
   },
   "dependencies": {
     "cdt-gdb-adapter": "^0.0.16-next",
+    "cdt-amalgamator": "^0.0.5",
```

Comment:
Please ensure changes to dependencies have corresponding changes to yarn.lock.

Reply:
The 0.0.6 version has been published that has these changes: https://www.npmjs.com/package/cdt-amalgamator/v/0.0.6

Reply:
I have updated the current version `0.0.6`. And added yarn.lock file